time stamp

float32 Fx
float32 Fy
float32 Fz
float32 Tx
float32 Ty
float32 Tz